debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java

```diff
@@ -1202,9 +1202,14 @@ ShrcConfig dbgShrcReadConfig(J9SharedClassConfigPointer sharedClassConfig, Print
 					/*  _ccHead -------------> ccNext ---------> ccNext --------> ........---------> ccTail
 					 * (top layer)         (middle layer)     (middle layer)      ........         (layer 0)
 					 */
-					SH_CompositeCacheImplPointer compositeCacheImpl = cacheMap._ccTail();
-					for (int tmplayer = 0; tmplayer < layer; tmplayer++) {
-						compositeCacheImpl = compositeCacheImpl._previous();
+					SH_CompositeCacheImplPointer compositeCacheImpl = null;
+					try {
+						compositeCacheImpl = cacheMap._ccTail();
+						for (int tmplayer = 0; tmplayer < layer; tmplayer++) {
+							compositeCacheImpl = compositeCacheImpl._previous();
+						}
+					} catch (NoSuchFieldError e) {
```

Comment:
This is not sufficient. When accessing an older core file from an openj9 VM, the type `SH_CompositeCacheImplPointer` will be dynamically generated without the `_ccTail()` method. This either needs to catch `NoSuchMethodError` or key off some other feature of old core files to avoid the call.

Reply:
Added code to catch `NoSuchMethodError`.